cloud_Azure/terraform/module/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Kentik integration Terraform module
 
-Module supporting management of Azure and Kentik resources required for flow log export from Azure to Kentik.  
+Terraform module, which creates Azure and Kentik resources, for Azure cloud integration in Kentik.
```

Comment:
The existing text is better (I think it is what I've suggested). The conflict just needs to be resolved the other way.

Reply:
Restoring.